docs/guides/Integration-Guides/tax-service-integration-guide/tax-services-specification.md

```diff
@@ -5,46 +5,68 @@ hidden: false
 
-### Checkout Configuration
+## Tax integration via Checkout API
 
-Synchronous tax integration can be activated or deactivated through a request to the Checkout Configuration API.
+You must activate the tax integration by configuring the `orderForm`, an object that stores contextual information about the order. This data is essential to the checkout process of the order.
 
-<img src="https://raw.githubusercontent.com/vtexdocs/dev-portal-content/main/images/tax-services-specification-0.svg"/>
+Check the flow of `orderForm` configuration below:
 
-First, it is necessary to GET the current `orderForm` settings from this endpoint:
-
-`https://{accountName}.{environment}.com.br/api/checkout/pvt/configuration/orderForm`
-
-Then, Checkout settings can be updated via a POST call to the same endpoint. The full request body can be found in the [Update orderForm configuration](https://developers.vtex.com/vtex-rest-api/reference/updateorderformconfiguration), as well as more details on that API. Here we will focus on the `taxConfiguration` object, which defines the External Tax Service settings:
+![Tax integration flow](https://raw.githubusercontent.com/vtexdocs/dev-portal-content/main/images/tax-service-flow-specification.png)
+To activate the tax integration, first it is necessary to get the current `orderForm` settings using the [Get `orderForm` configuration](https://developers.vtex.com/docs/api-reference/checkout-api#get-/api/checkout/pvt/configuration/orderForm?endpoint=get-/api/checkout/pvt/configuration/orderForm).
+In the request response, the `taxConfiguration` object has the tax information that must be updated.
 
 ```json
-  "taxConfiguration": {
-        "url": "https://accountname.myvtex.com/tax-service/order-tax",
+    "taxConfiguration": {
+        "url": "https://{accountName}.myvtex.com/tax-service/order-tax",
         "authorizationHeader": "99b9935b048dfd86893d0bf9gas628849",
-    }
+	      "appId": "tradeincart",
+        ...
+    },
+  "isMarketplaceResponsibleForTaxes": false,
+   ...
 ```
 
-The most important data in this object is the `url`. This is the endpoint URL that the Checkout will query to receive the calculated taxes. In other words, this is the external API endpoint of the tax tool.
+Then, Checkout settings can be updated via [Update orderForm configuration](https://developers.vtex.com/docs/api-reference/checkout-api#post-/api/checkout/pvt/configuration/orderForm?endpoint=post-/api/checkout/pvt/configuration/orderForm) endpoint.
+
+The most important data in the `taxConfiguration` object is the `url`. This is the external API endpoint of the tax tool that the Checkout will query to receive the calculated taxes. Here is an endpoint example from [Avalara](https://www.avalara.com/us/en/index.html/):
```

Comment:
It would be way more visual and easier to navigate if these properties to be changed, currently described in paragraphs, were inside a table containing Property Name, Description, Default Value and Example.